### PR TITLE
Bug 1812517 - Update error pages for https and expired certificate. Modified the texts and added a link.

### PR DESCRIPTION
**Pull request by iorgamgabriel**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1812517

### CODE REVIEW

**Review by Mugurell — COMMENTED** [review]
Looks good but I want to double check the need for new strings in A-C and how could we ease maintenance of the newly added `css` file.

File: android-components/components/browser/errorpages/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -24,23 +24,39 @@
     ]]></string>
 
     <!-- The document title and heading of the error page shown when a website sends has an invalid or expired SSL certificate. -->
-    <string name="mozac_browser_errorpages_security_bad_cert_title">Secure Connection Failed</string>
+    <string name="mozac_browser_errorpages_security_bad_cert_title" moz:RemovedIn="111" tools:ignore="UnusedResources">Secure Connection Failed</string>
+
+    <!-- The document title and heading of the error page shown when a website sends has an invalid or expired SSL certificate. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_title2">Connection not secure</string>
```
[the remark this answers is not in the document]
Comment (iorgamgabriel) [reply android-components/components/browser/errorpages/src/main/res/values/strings.xml]: Done

File: focus-android/app/src/main/assets/error_style.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  display: none;
+}
+
+#badCertAdvancedPanel {
+  display: none;
+}
+
+a {
+  color: white;
+}
```
[the remark this answers is not in the document]
Comment (iorgamgabriel) [reply focus-android/app/src/main/assets/error_style.css:210]: Done

**Comment by iorgamgabriel** [comment]
https://user-images.githubusercontent.com/8118371/216629879-934fcdeb-3dca-49bd-87a7-f4ac367ececc.mp4

**Review by Mugurell — APPROVED** [review]
LGTM, thanks!

**Pull request merged**